<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IzvestavanjeOSredstvimaD2</class>
 <widget class="QDialog" name="IzvestavanjeOSredstvimaD2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Izvestavanje o sredstvima</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mesec:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbMesec">
       <item>
        <property name="text">
         <string>Svi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Januar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Februar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maj</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jul</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Avgust</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Septembar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oktobar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Novembar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Decembar</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Godina:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbGodina"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tvIzvestaj"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnPrikazi">
       <property name="text">
        <string>Prikaži</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSacuvaj">
       <property name="text">
        <string>Sačuvaj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOdstampaj">
       <property name="text">
        <string>Odštampaj</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnIzadji">
       <property name="text">
        <string>Izađi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
